example/readFromString/readFromString.cpp

```diff
@@ -0,0 +1,33 @@
+#include "json/json.h"
+#include <iostream>
+/*
+parse a string to Value object with CharReaderBuilder class or Reader class
+$g++ readFromString.cpp -ljsoncpp -std=c++11 -o readFromString
+$./readFromString
+colin
+20
+*/
+int main() {
+  std::string strRes = R"({"Age": 20, "Name": "colin"})";
+  auto nLen = (int)strRes.length();
+  JSONCPP_STRING err;
+  Json::Value root;
+
+#if 0 // old way
+    Json::Reader myreader;
+    myreader.parse(strRes,root);
+#else // new way
+  Json::CharReaderBuilder builder;
+  std::unique_ptr<Json::CharReader> const reader(builder.newCharReader());
```

Comment:
Prefer `const foo` to `foo const`

Reply:
Do you mean `const std::unique_ptr<Json::CharReader> reader(builder.newCharReader());
`?